packages/search/src/search.tsx

```diff
@@ -310,7 +311,7 @@ class Search extends React.Component< Props, State > {
 
 	render() {
 		const searchValue = this.state.keyword;
-		const placeholder = this.props.placeholder || __( 'Search…' );
+		const placeholder = this.props.placeholder || this.props.__( 'Search…', __i18n_text_domain__ );
```

Comment:
I was wondering why we need the textdomain argument now.

Reply:
I think we need it in case the search component is used outside of the `default` context. And in this case we're receiving `__` from `withI18n` so we need to still supply a context, right?